front/app/modules/commercial/customizable_navbar/admin/components/NavbarTitleField.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import InputMultilocWithLocaleSwitcher from 'components/HookForm/InputMultilocWithLocaleSwitcher';
+import { InjectedIntlProps } from 'react-intl';
+import { injectIntl } from 'utils/cl-intl';
+import messages from './messages';
+import usePage from 'hooks/usePage';
+import { isNilOrError } from 'utils/helperUtils';
+import { POLICY_PAGES, TPolicyPage } from 'services/pages';
+
+type Props = {
+  pageId: string | null;
+};
+
+const NavbarTitleField = ({
+  pageId,
+  intl: { formatMessage },
+}: Props & InjectedIntlProps) => {
+  const page = usePage({ pageId });
+  if (
+    isNilOrError(page) ||
+    POLICY_PAGES.includes(page.attributes.slug as TPolicyPage)
```

Comment:
This is actually not correct. With `as TPolicyPage` you're telling TypeScript "each instance of page.attributes.slug is a TPolicyPage".

I found it pretty hard to do this with TS, but I think I found [n acceptable solution](https://github.com/CitizenLabDotCo/citizenlab/compare/CL-1308-one-more-attempt...CL-1308-review-24/8?expand=1). I had a typo in enum (wrapped 'policy-page' with another set of double quotes: "'policy-page'") and it signaled an error, which is nice.

Reply:
Feel free to merge my suggestion.